content/en/docs/collector/configuration.md

```diff
@@ -490,21 +489,26 @@ extensions:
     size_mib: 512
 ```
 
-## Service
+> For detailed extension configuration, see the
+> [extension README](https://github.com/open-telemetry/opentelemetry-collector/blob/main/extension/README.md).
+
+## Service section {#service}
 
 The service section is used to configure what components are enabled in the
 Collector based on the configuration found in the receivers, processors,
 exporters, and extensions sections. If a component is configured, but not
-defined within the service section then it is not enabled. The service section
-consists of three sub-sections:
+defined within the service section, then it's not enabled.
+
+The service section consists of three subsections:
 
-- extensions
-- pipelines
-- telemetry
+- `extensions:`
+- `pipelines:`
+- `telemetry:`
```

Comment:
Could we drop the colons here? I think that it is sufficient to name the YAML key. WDYT?

Reply:
Done! Switched to uppercase items so as to turn the list into merely structural, rather than YAML related.